vcx/ci/Jenkinsfile

```diff
@@ -97,10 +97,10 @@ def android() {
                 //Package .so files into .aar 
                 packageAndroid(rust)
 
-                if (env.BRANCH_NAME == "master") {
+                // if (env.BRANCH_NAME == "master") {
                     //Publish package on aptly
                     publishAndroid(rust)
-                }
+                // }
             } catch (Exception ex) {
```

Comment:
Naga, does this need to be uncommented so that it only publishes the package to aptly for the master branch?

Reply:
this forces the .aar file to be built on git branch. Before it is merged into master. This is for test code